tools/get-app-size.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# set safer error and list handling
+set -euo pipefail
+IFS=$'\n\t'
```

Comment:
It's just the regular newline and tab. The idea is to remove space from the default IFS to avoid issues when iterating lists of filenames etc.

Setting IFS here doesn't really matter since it's set later when used, but I think it's a good practice to keep my feet safer.